modules/nw-multinetwork-sriov.adoc

```diff
@@ -40,247 +62,386 @@ and device ID 0x1015
+then installs with an *InstallSucceeded* message, you can safely ignore
+the *Failed* message.
+====
+
 +
-----
-oc create -f sriov-device-plugin.yaml
-----
+If the operator does not appear as installed, to troubleshoot further:
 
-. Create a YAML file for the `sriov-cni` DaemonSet with the following contents:
 +
-[source,yaml,subs="attributes"]
+* Switch to the *Catalog* → *Operator Management* page and inspect
+the *Operator Subscriptions* and *Install Plans* tabs for any failure or errors
+under *Status*.
+* Switch to the *Workloads* → *Pods* page and check the logs in any Pods in the
+`sriov-network-operator` projects that are reporting issues.
+
+== Discover SR-IOV network devices
+
+After the SR-IOV network Operator has been install successfully. The operator
+will try to discover all the SR-IOV capable network devices on worker nodes.
+User can find those information from the `SriovNetworkNodeState` Custom 
+Resources, which are generated and updated by the operator automatically. 
+
+One CR is created for each worker node, and shares the same name as the node. In
+the interface list, you can find the information of the network devices.  
+
+You should never have to modify this CRD, nor the CRs. 
+
+The following is an example of a typical Custom Resource for
+`SriovNetworkNodeState`.
+
+[source,yaml]
 ----
-kind: DaemonSet
-apiVersion: apps/v1
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodeState
 metadata:
-  name: sriov-cni
-  namespace: openshift-sriov
-  annotations:
-    kubernetes.io/description: |
-      This daemon set launches the SR-IOV CNI plugin on SR-IOV capable worker nodes.
+  creationTimestamp: "2019-08-27T06:01:36Z"
+  generation: 1
+  name: node-25
+  namespace: sriov-network-operator
+  ownerReferences:
+  - apiVersion: sriovnetwork.openshift.io/v1
+    blockOwnerDeletion: true
+    controller: true
+    kind: SriovNetworkNodePolicy
+    name: default
+    uid: 20c3c0f9-c890-11e9-91f7-0028ccd628ee
+  resourceVersion: "2639363"
+  uid: 2103a9fe-c890-11e9-91f7-0028ccd628ee
 spec:
-  selector:
-    matchLabels:
-      app: sriov-cni
-  updateStrategy:
-    type: RollingUpdate
-  template:
-    metadata:
-      labels:
-        app: sriov-cni
-        component: network
-        type: infra
-        openshift.io/component: network
-    spec:
-      nodeSelector:
-        kubernetes.io/os: linux
-      tolerations:
-      - operator: Exists
-      serviceAccountName: sriov-cni
-      containers:
-      - name: sriov-cni
-        image: quay.io/openshift/{image-prefix}-sriov-cni:v4.0.0
-        securityContext:
-          privileged: true
-        volumeMounts:
-        - name: cnibin
-          mountPath: /host/opt/cni/bin
-      volumes:
-        - name: cnibin
-          hostPath:
-            path: /var/lib/cni/bin
-----
-
-. Run the following command to create the `sriov-cni` DaemonSet:
-+
-----
-$ oc create -f sriov-cni.yaml
+  dpConfigVersion: d41d8cd98f00b204e9800998ecf8427e
+status:
+  interfaces:
```

Comment:
Shall we have a explanation for `interfaces` field, followed by this example?
for example, all the properties are for physical functions.
We have `vendorID` and `deviceID` mentioned in several places in the doc, but they may refer to PF or VF IDs depending on whether they are in a `status` field of `SriovNetworkNodeState` CR or in the `spec` field of `SriovNetworkNodePolicy` CR.

Reply:
The field under `interfaces` is quite straightforward. User shall be able to know what it is through the name.